code/game/objects/random/biogenerator_randoms.dm

```diff
@@ -0,0 +1,82 @@
+// This list of randoms is intended for use in the biogenerator.
+// These ones are sanitized of anything funky compared to other random item lists
+
+/obj/random/biogenerator/backpack
+	name = "random backpack"
+	icon = 'icons/obj/storage/backpack.dmi'
+	icon_state = "backpack"
+	spawnlist = list(
+		/obj/item/storage/backpack,
+		/obj/item/storage/backpack/medic,
+		/obj/item/storage/backpack/security,
+		/obj/item/storage/backpack/industrial,
+		/obj/item/storage/backpack/hydroponics,
+		/obj/item/storage/backpack/pharmacy,
+		/obj/item/storage/backpack/satchel,
+		/obj/item/storage/backpack/toxins,
+		/obj/item/storage/backpack/satchel/leather,
+		/obj/item/storage/backpack/satchel/eng,
+		/obj/item/storage/backpack/satchel/med,
```

Comment:
This being a random list is fine, but it's like, way too expansive and random.
It should only include plain backpacks/satchels/duffelbags/messengerbags/etc.
Plain as in, not departmental.

Both cause the biogenerator can be used by offships and the likes.
And also cause it'd require generating a lot of useless backpacks if you want to give someone a generic one.